### PR TITLE
Adding imNetwork to IP Activity

### PR DESCRIPTION
**Pull request by yaronMSFT**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Insights/IP/IP_SessionCounts.yaml [hunk 1]
```diff
@@ -93,6 +93,12 @@ BaseQuery: |
   //| where not(DeviceAction has_any ('built','deny', 'denied', 'rst', 'blocked', 'teardown'))
   | extend Direction = iff(CommunicationDirection !in~ ('outbound','0') or CommunicationDirection !in~ ('inbound','1'), 'NotAvailable', CommunicationDirection)
   | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), InboundCount = countif(Direction in~ ('Inbound','1')), OutboundCount = countif(Direction in~ ('Outbound', '0')), UnknownDirection = countif(Direction =~ 'NotAvailable') by IPAddress = DestinationIP, Type = strcat(Type,':', DeviceVendor,'-', DeviceProduct), RemoteIPAddress = SourceIP, Direction, SentBytes = tolong(SentBytes), ReceivedBytes = tolong(ReceivedBytes)
+  ),
+  (
+  imNetworkSession(starttime=start_time, endtime=end_time, dstipaddr=Ip_Address) // , dvcaction='Allow')
+  | where SrcIpAddr != DstIpAddr
```
Comment (oshezaf) [inline Insights/IP/IP_SessionCounts.yaml:99]: How do you think unknown direction would work?
How do we handle duplicates?

**Comment by oshezaf** [comment]
@YaronFruchtmann : note that I commented on the PR.

File: Insights/IP/IP_SessionCounts.yaml (hunk 1 above)
Comment (shainw) [reply Insights/IP/IP_SessionCounts.yaml:99]: Direction: I think unknown direction needs to be used on conjunction with the source data type.  In other words, not all data types provide direction or it is implied based on the data type.  I think we make that clear in the result to help the user.  Thoughts?

Duplicates: Seems like duplicates is a loaded term here.  For example, duplicates of an IP between a variety of logs is expected, but I don't think we should obfuscate that.  I think we could do a makeset() or pack() or the like on the Type column to aggregate but still show that this IP showed up in x,y,z data types.  Thoughts?

**Comment by shainw** [comment]
@YaronFruchtmann and @oshezaf - I also commented. ;)

**Comment by oshezaf** [comment]
Comments from Yaron and my conversation: 
- First, not a high priority.
- Will wait to inNetwork LA solution functions deployment

Duplication was meant in the context of ASIM and direct source use.

**Comment by shainw** [comment]
@YaronFruchtmann and @oshezaf - do we want to revisit this one now that ASIM is released?

**Comment by v-mchatla** [comment]
Hi @oshezaf
Can we close this and reopen whenever required as you have labelled it as differed.
Thanks

**Comment by oshezaf** [comment]
As I wrote, those are intentionally kept open as we intend to get back to them soon.

**Pull request closed without merging**